include/hpp/fcl/narrowphase/narrowphase.h

```diff
@@ -309,7 +309,7 @@ namespace fcl
     GJKSolver()
     {
       gjk_max_iterations = 128;
-      gjk_tolerance = 1e-5;
+      gjk_tolerance = 1e-4;
```

Comment:
A tolerance of `1e-6` on hppfcl convergence criterion corresponds to `1e-4` on the duality-gap